iam/sc-enduser-iam.yaml

```diff
@@ -30,12 +30,10 @@ Resources:
         Statement:
           - Effect: Allow
             Principal:
-              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
-              # Synapse OIDC identity provider
-              Federated: !Sub 'arn:aws:iam::${AWS::AccountId}:oidc-provider/repo-prod.prod.sagebase.org/auth/v1'
```

Comment:
I think removing this breaks the ConnectionURI, the ability to do an SSH connection using the AWS web console.  The error when clicking on the `ConnectionURI` link is: 
```
User: arn:aws:sts::465877038949:assumed-role/ServiceCatalogEndusers/3377358 is not authorized to perform: ssm:StartSession on resource: arn:aws:ec2:us-east-1:465877038949:instance/i-02b1c202c36e86c93
```
It causes the `EndUserRoleId` to be null at https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/iam/sc-enduser-iam.yaml#L102 

The EC2 templates rely on the `EndUserRoleId` info here https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/ec2/sc-ec2-linux-jumpcloud-v1.0.0.yaml#L430

Reply:
I checked scipooldev which has a manually implemented version of this change, and I *was* able to use a ConnectionUri.